gio/src/socket.rs

```diff
@@ -258,6 +556,114 @@ impl<O: IsA<Socket>> SocketExtManual for O {
         }
     }
 
+    fn send_message<P: IsA<SocketAddress>, C: IsA<Cancellable>>(
+        &self,
+        address: Option<&P>,
+        vectors: &[&[u8]],
```

Comment:
Why not have this take a `&[OutputVector]`?

Reply:
Missed this one, It takes `OutputVector` now :)